core/server/src/main/java/alluxio/worker/DefaultAlluxioWorker.java

```diff
@@ -102,50 +100,46 @@
   private AtomicReference<Long> mWorkerId;
 
   /**
-   * Constructs a {@link DefaultAlluxioWorker}.
+   * Creates a new instance of {@link DefaultAlluxioWorker}.
+   *
+   * @throws Exception if the creation fails
    */
-  public DefaultAlluxioWorker() {
+  public DefaultAlluxioWorker() throws Exception {
```

Comment:
Either throw typed exceptions (if we plan to handle them) or use Throwables.propagate (if we will not be able to handle them)?